scheduler/scheduler-api/src/main/java/org/ow2/proactive/scheduler/synchronization/SynchronizationWrapper.java

```diff
@@ -0,0 +1,248 @@
+    }
+
+    @Override
+    public <T extends Serializable> T putIfAbsent(String channel, String key, T value)
+            throws InvalidChannelException, IOException {
+        return internalAPI.putIfAbsent(originator, taskId, channel, key, value);
+    }
+
+    @Override
+    public <T extends Serializable> boolean remove(String channel, String key, T value)
+            throws InvalidChannelException, IOException {
+        return internalAPI.remove(originator, taskId, channel, key, value);
+    }
+
+    @Override
+    public <T extends Serializable> boolean replace(String channel, String key, T oldValue, T newValue)
+            throws InvalidChannelException, IOException {
+        return internalAPI.replace(originator, taskId, channel, key, oldValue, newValue);
+    }
+
+    @Override
+    public <T extends Serializable> T replace(String channel, String key, T value)
+            throws InvalidChannelException, IOException {
+        return internalAPI.replace(originator, taskId, channel, key, value);
+    }
+
+    @Override
+    public <T extends Serializable> T computeIfAbsent(String channel, String key, String mappingFunction)
+            throws InvalidChannelException, CompilationException, IOException {
+        return internalAPI.computeIfAbsent(originator, taskId, channel, key, mappingFunction);
+    }
+
+    @Override
+    public <T extends Serializable> T computeIfPresent(String channel, String key, String remappingFunction)
+            throws InvalidChannelException, CompilationException, IOException {
+        return internalAPI.computeIfPresent(originator, taskId, channel, key, remappingFunction);
+    }
+
+    @Override
+    public <T extends Serializable> T compute(String channel, String key, String remappingFunction)
+            throws InvalidChannelException, CompilationException, IOException {
+        return internalAPI.compute(originator, taskId, channel, key, remappingFunction);
+    }
+
+    @Override
+    public <T extends Serializable> T merge(String channel, String key, T value, String remappingFunction)
+            throws InvalidChannelException, CompilationException, IOException {
+        return internalAPI.merge(originator, taskId, channel, key, value, remappingFunction);
+    }
+
+    @Override
+    public <T extends Serializable> PredicateActionResult<T> conditionalCompute(String channel, String key,
+            String predicate, String thenRemappingFunction)
+            throws InvalidChannelException, CompilationException, IOException {
+        return internalAPI.conditionalCompute(originator, taskId, channel, key, predicate, thenRemappingFunction);
+    }
+
+    @Override
+    public <T extends Serializable> PredicateActionResult<T> conditionalCompute(String channel, String key,
+            String predicate, String thenRemappingFunction, String elseRemappingFunction)
+            throws InvalidChannelException, CompilationException, IOException {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)